spring-petclinic-vets-service/pom.xml

```diff
@@ -42,11 +46,13 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
+            <exclusions>
```

Comment:
In order to exclude the junit:junit dependency a single time, we could centralize the exclusion into the <dependenciesManagement> section of the parent POM